### DISCUSSION TITLE
Can't login via email

### DISCUSSION THREAD

**Discussion by salario07** [codemagic-ci-cd/codemagic-docs#2042] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1- Go to the login page
2- Enter the account email and request an authentication key.
3- It says that Code Magic sent an email to us but we are not receiving it or receiving it when it is expired.

### Expected results

1- Receive the authentication key in my email inbox immediately after requesting.
2- Be able to log in using the key

### Actual results

1- No email has been received
2- Email has been received when the authentication key is expired
3- After some time (for example one hour) all the codes we requested arrive together but they are all expired.
4- Not able to log in.

**Comment by himesh-cm** [comment]
Hi @salario07, Can you please share the team id so i can look into the same?

**Reply by himesh-cm** [reply]
I requested a code from the login page and logs show it was sent. Can you confirm receipt?

**Reply by salario07** [reply]
Sorry, I have no access to team ID.
Did you request it with our email?
I have to check with the company manager to see if we get a code or not. I will tell you here.

**Reply by himesh-cm** [reply]
Yes, via email, the one you shared earlier.

**Reply by mikhail-tokarev** [reply]
@salario07 please never share your code

**Reply by salario07** [reply]
Ok, thanks. Do you manage to login? Do you have any clue why we can't login?

**Reply by himesh-cm** [reply]
@salario07, You should be able to log in via the code, Please let me know if you still face any issues.

**Reply by salario07** [reply]
Ok I will try again. Thanks for your quick response.

**Comment by salario07** [comment]
Still have the same issue, Can someone increase the token expiry duration to one hour for our account? We receive the code when it is expired.
@himesh-cm 
@mikhail-tokarev

**Reply by salario07** [reply]
Even when we receive and enter code in 7 mins, it said expired or missing.
`{"error":"NOT_FOUND","message":"Authentication key has expired or is missing"}`

I think there is a problem with our account.

**Reply by dtrdic** [reply]
@salario07 do you still have login problems?

**Comment by mohammadprgrm20123** [comment]
Hi I have this issue too

@himesh-cm 
@mikhail-tokarev

**Reply by dtrdic** [reply]
@mohammadprgrm20123 do you still have login problems?

**Reply by mohammadprgrm20123** [reply]
No, I log in  

Thanks for your supporting
